lib/android-tools-info.ts

```diff
@@ -186,6 +139,51 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		return null;
 	}
 
+	@cache()
+	public validateAndroidHomeEnvVariable(options?: { showWarningsAsErrors: boolean }): boolean {
```

Comment:
The cache decorator has tests [here](https://github.com/telerik/mobile-cli-lib/blob/master/test/unit-tests/decorators.ts#L421-L494)
The `androidHome` will not be changed between executions of the method (I've also change androidHome to be only getter). 

Regarding tests for the whole androidToolsInfo - we'll move it to `nativescript-doctor` package very soon and we'll add tests there.